Chart.yaml

```diff
@@ -0,0 +1,16 @@
+# -------------------------------------------------------------------------------------
+#
+# Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com). All Rights Reserved.
```

Comment:
Shall we change the license to Apache 2. WSO2 public deployment artefacts are released under Apache. 

Eg: https://github.com/wso2/apk/blob/main/helm-charts/templates/idp/authenticationEndpoint-domain-api-httproute.yaml